certbot-ci/certbot_integration_tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+"""
+General conftest for pytest execution of all integration tests lying
+in the certbot_integration tests package.
+As stated by pytest documentation, conftest module is used to set on
```

Comment:
link?

Reply:
https://docs.pytest.org/en/latest/reference.html#hook-reference

Reply:
Added to the module doc.